uint8 game_type
uint8 game_progress
uint16 stage_remain_time
uint64 SyncTimeStamp